test/e2etest/util/util.go

```diff
@@ -0,0 +1,129 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/gardener/test-infra/pkg/util"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func DownloadFile(url, dir string) (filePath string, err error) {
+	request, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+	fileName := path.Base(request.URL.Path)
+	filePath = filepath.Join(dir, fileName)
+	fileData, err := util.DownloadFile(&http.Client{}, url)
+	if err != nil {
+		return "", err
+	}
+	if err := ioutil.WriteFile(filePath, fileData, 0777); err != nil {
+		return "", err
+	}
+	return filePath, nil
+}
+
+func UniqueStrings(stringSlice []string) []string {
+	keys := make(map[string]bool)
+	list := []string{}
+	for _, entry := range stringSlice {
+		if _, value := keys[entry]; !value {
+			keys[entry] = true
+			list = append(list, entry)
+		}
+	}
+	return list
+}
+
+func GetEnv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
+func RunCmd(command, execPath string) error {
+	separator := " "
+	parts := strings.Split(command, separator)
+
+	head := parts[0]
+	args := parts[1:len(parts)]
+	cmd := exec.Command(head, args...)
+	if execPath != "" {
+		cmd.Dir = execPath
+	}
+
+	//	Sanity check -- capture stdout and stderr:
+	var out bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &stderr
+
+	err := cmd.Run()
+
+	//	Output our results
+	if out.String() != "" {
+		log.Info(out.String())
+	}
+	if stderr.String() != "" {
+		log.Error(fmt.Println(stderr.String()))
+	}
+
+	return err
+}
+
+// Contains tells whether a contains x.
+func Contains(a []string, x string) bool {
+	for _, n := range a {
+		if x == n {
+			return true
+		}
+	}
+	return false
+}
+
+func SilentStrToInt(s string) int {
```

Comment:
why do you want to silenlty parse string to int?
if the string is not parsable then 0 is returned which acts like a valid result.